declarations.d.ts

```diff
@@ -1,5 +1,6 @@
 declare module "*.json";
 declare module "webpack-cli";
+declare module "webpack-command";
```

Comment:
Looks like we don't need `webpack-cli` and `webpack-command` now, should i remove this?